tools/StartDistributedTrace/Program.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Globalization;
+using System.Net.Http;
+using Datadog.Trace;
+using Datadog.Trace.Headers;
+
+namespace StartDistributedTrace
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.WriteLine($"Usage: {nameof(StartDistributedTrace)} <url>");
+                return;
+            }
+
+            string url = args[0];
+            Console.WriteLine($"[HttpClient] sending request to {url}");
+
+            using (var scope = Tracer.Instance.StartActive("http.request"))
+            using (var client = new HttpClient())
+            {
+                // Set basic span information for HTTP operations
+                var span = scope.Span;
+                span.Type = SpanTypes.Http;
+                span.SetTag(Tags.HttpMethod, "GET");
+                span.SetTag(Tags.HttpUrl, url);
+
+                // Set distributed tracing headers
+                client.DefaultRequestHeaders.Add(HttpHeaderNames.TraceId, span.TraceId.ToString(CultureInfo.InvariantCulture));
+                client.DefaultRequestHeaders.Add(HttpHeaderNames.ParentId, span.SpanId.ToString(CultureInfo.InvariantCulture));
+                client.DefaultRequestHeaders.Add("upstream-service", nameof(StartDistributedTrace));
+
+                // Send HTTP request
+                using (var responseMessage = client.GetAsync(url).Result)
```

Comment:
We can use `await` here (and below) if we make `Main()` `async`.
I know that getting `Task.Result` is safe in some contexts, but I would prefer to avoid if possible 😅

Reply:
Good feedback, applied in 3be3025d1ae6cd911e9b3d45d87669396a023d49